### PR TITLE
Improve wynncraft team nametags

### LINKED ISSUE #144: People with [Art] rank do not have the Wynncraft CT tag

### PR DESCRIPTION
**Pull request by BurstingF**
This makes the nametags for the wynncraft teams (such as Moderator, Admin, GM) based on the name of the scoreboard team the player has assigned by wynncraft. 

For Moderators and Admins I don't think anything changes in appearance. People that originally had "Wynncraft CT" will now get their specific rank such as GM or Builder. 

This might also fix [Issue#144](https://github.com/Wynntils/Wynntils/issues/144) but my guess is that [Issue#144](https://github.com/Wynntils/Wynntils/issues/144) is happening because the specific player with the art rank doesn't have a scoreboard team or the art rank in general doesn't have a scoreboard team.

### CODE REVIEW

**Review by HeyZeer0 — CHANGES_REQUESTED** [review]
as safe as it sounds, this actually is called every single rendering tick and creating a new object every time that there's a staff member could cause a massive memory leak. That's the only issue ^^

**Comment by BurstingF** [comment]
That indeed sounds like an issue I didn't think of. 
I think this new commit should fix the issue you described.

File: src/main/java/com/wynntils/modules/utilities/managers/NametagManager.java [hunk 1]
```diff
@@ -69,7 +71,7 @@ public static boolean checkForNametags(RenderLivingEvent.Specials.Pre e) {
 
             //wynncraft teams (Admin, Moderator, GM, Builder, etc.)
             if (entity.getTeam() != null && !(entity.getTeam().getName().equalsIgnoreCase("all") || entity.getTeam().getName().equalsIgnoreCase("afk"))) {
```
Comment (HeyZeer0) [inline src/main/java/com/wynntils/modules/utilities/managers/NametagManager.java:73]: ``entity.getTeam().getName().equalsIgnoreCase("afk")`` is this if cause right? Seems like
``entity != null && (!name.equals("all") || name.equals("afk"))``
So you're considerating just afk as a wynncraft team? 

It should probably be
``if (entity.getTeam() != null && !entity.getTeam().getName().equalsIgnoreCase("all") && !entity.getTeam().getName().equalsIgnoreCase("afk")) {``

**Review by HeyZeer0 — CHANGES_REQUESTED** [review]

File: src/main/java/com/wynntils/modules/utilities/managers/NametagManager.java (hunk 1 above)
Comment (BurstingF) [reply src/main/java/com/wynntils/modules/utilities/managers/NametagManager.java:73]: `entity.getTeam() != null && (!name.equals("all") || name.equals("afk"))` is not actually how it is written down at the moment. 

Currently it is written down as `entity.getTeam() != null && !(name.equals("all") || name.equals("afk"))` which is equivalent to `entity.getTeam() != null && !name.equals("all") && !name.equals("afk")`. 

I can still change it if you think !A && !B is more readable than !(A || B).

**Pull request merged**